rca/static_src/javascript/programmes/components/ProgrammesExplorer.js

```diff
@@ -28,6 +29,12 @@ const ProgrammesExplorer = ({ searchLabel, categories }) => {
     return (
         <>
             <SearchForm searchQuery={searchQuery} label={searchLabel} />
+            {/* Study mode toggler placeholder */}
+            <ToggleSwitch
+                ariaLabel="Programme study mode"
+                labelOne="Full-time"
+                labelTwo="Part-time"
+            />
```

Comment:
According to the design, this component shouldn't be here. I just placed it here as it was easier and quicker for me as I tested things out.